research_notebooks/explain_documentation/01_load_hummingbot_docs.ipynb

```diff
@@ -0,0 +1,728 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Loading Hummingbot\n",
+    "\n",
+    "We are going to use a Generic Loader and get the files from the scripts folder of the Hummingbot repository.\n",
+    "Also, we are using a LanguageParser to parse the files as Python code."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import time\n",
+    "\n",
+    "from dotenv import load_dotenv\n",
+    "from pprint import pprint\n",
+    "\n",
+    "load_dotenv()\n",
+    "\n",
+    "os.environ[\"OPENAI_API_KEY\"]\n",
+    "# you should see the api key if the .env file is loaded correctly"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "outputs": [],
+   "source": [
+    "from custom_loaders.mkdocs_site_loader import MkDocsSiteLoader\n",
```

Comment:
when I ran `jupyter notebook` from the root directory, I got a message saying that it couldn't find the custom loaders module

<img width="1175" alt="Screen Shot 2023-12-05 at 12 41 40 PM" src="https://github.com/hummingbot/ai-assistant/assets/1795030/5e2bfd6f-c9fb-449c-903a-6d48f2ae4828">

Reply:
mmm I'm running the notebook from pycharm, I should look how to make Jupyter know which is the root directory